app/assets/javascripts/users.coffee

```diff
@@ -0,0 +1,10 @@
+$ ->
+  $(document.body).on "click", "[data-action]", (e) ->
```

Comment:
It's not always necessary to put values in data-* attribute, this can sometimes be substituted for class selectors and can be extended to actions where the class selector is not necessary. This can also substitute the use of three routes `(beard, shave..)`, with one, but i didn't implemented that.